api/mesh/options.proto

```diff
@@ -40,6 +40,13 @@ message KumaResourceOptions {
 
   // WsAdmin whether this entity requires admin auth to access these endpoints
   bool ws_admin = 11;
+
+  // HasCustomKumaCtlRender whether we should use a custom function to render
+  // entities in kumactl
+  bool has_custom_kumactl_render = 12;
+
+  // SkipKumactlGet don't generate get commands for kumactl
+  bool skip_kumactl_get = 13;
```

Comment:
I think we need a better-name for this flag. I think it means that this type is not visible at all in kumactl (no get or delete support)